app/controllers/application_controller.rb

```diff
@@ -0,0 +1,46 @@
+class ApplicationController < ActionController::API
+    before_action :authorized
+
+    def authorized
+        render json: {message: "Please log in"},status: :unauthorized unless logged_in?
+    end
+    def logged_in?
+    !!logged_in_user
+    end
+
+
+    def logged_in_user
+    if decoded_token
+        user_id=decoded_token[0]['user_id']
+        @user=User.find_by(id: user_id)
+    end
+end
+
+
+    def decodeds_token
+        if auth_header
+            token=auth_header.split(' ')[1]
+            begin
+                JWT.decode(token, 'secret',true,algorith: 'HS256')
```

Comment:
token, 'secret',true,algorith: 'HS256' make sure after coma space will there in everywhere